.github/workflows/test.yml

```diff
@@ -19,18 +19,22 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      # Add or replace dependency steps here
```

Comment:
Not a huge deal, but something this file should really be in its own PR

Reply:
@aarondufall Fair point, in this case however if I had merged this PR without this `yarn install` step, I would be introducing a failure to the pipeline. I think merging just this change, before the actions implemented in this PR wouldn't have installed any visible/used dependencies? :thinking: